src/Microsoft.DotNet.Darc/src/Darc/Operations/GetDependencyGraphOperation.cs

```diff
@@ -193,20 +277,60 @@ private string GetGraphVizNodeName(DependencyGraphNode node)
             return GetSimpleRepoName(node.RepoUri).Replace("-", "") + node.Commit;
         }
 
-        private void LogGraphViz(DependencyGraph graph)
+        private async void LogGraphViz(DependencyGraph graph)
         {
-            Console.WriteLine("digraph repositoryGraph {");
-            Console.WriteLine("    node [shape=record]");
-            foreach (DependencyGraphNode node in graph.Nodes)
+            string directory = Path.GetDirectoryName(_options.GraphVizOutputFile);
+            if (!Directory.Exists(directory))
             {
-                Console.WriteLine($"    {GetGraphVizNodeName(node)}[label=\"{GetSimpleRepoName(node.RepoUri)}\\n{node.Commit.Substring(0, 5)}\"];");
-                foreach (DependencyGraphNode childNode in node.Children)
+                Directory.CreateDirectory(directory);
+            }
+
+            using (StreamWriter writer = new StreamWriter(_options.GraphVizOutputFile))
+            {
+                await writer.WriteLineAsync("digraph repositoryGraph {");
+                await writer.WriteLineAsync("    node [shape=record]");
+                foreach (DependencyGraphNode node in graph.Nodes)
                 {
-                    Console.WriteLine($"    {GetGraphVizNodeName(node)} -> {GetGraphVizNodeName(childNode)}");
+                    string buildString = "";
+                    if (node.ContributingBuilds != null && node.ContributingBuilds.Any())
+                    {
+                        Build newestBuild = node.ContributingBuilds.OrderByDescending(b => b.DateProduced).First();
+                        buildString = $"\\n{newestBuild.DateProduced.Value.ToLocalTime().ToString("g")}";
```

Comment:
In a file, localtime is confusing, especially without the time zone specified. Make it UTC and probably include the timezone so it's clear it's UTC.

Reply:
Updated